### PR TITLE
Rename template used by workload opeartor

### PR DESCRIPTION
**Pull request by pacospace**
Related-To: https://github.com/thoth-station/adviser/pull/769
Signed-off-by: Francesco Murdaca <fmurdaca@redhat.com>

### CODE REVIEW

File: thoth/common/openshift.py [hunk 1]
```diff
@@ -1633,7 +1633,7 @@ def get_adviser_template(self) -> Dict[str, Any]:
                 "Infra namespace is required to gather adviser template when running it"
             )
 
-        return self._get_template("template=adviser")
+        return self._get_template("template=adviser-argo")
```
Comment (fridex) [inline thoth/common/openshift.py]: `if self.use_argo`?

Comment (pacospace) [reply thoth/common/openshift.py]: right, `adviser-workload-operator`

Comment (pacospace) [reply thoth/common/openshift.py]: We can switch the other way around, as Argo will be the default, as suggested by @goern. WDYT?

Comment (fridex) [reply thoth/common/openshift.py]: I think we missed each other:

```python
if self.use_argo:
        return self._get_template("template=adviser")
else:
        return self._get_template("template=adviser-workload-operator")
```

To stay compatible with the env variable switch.

Comment (pacospace) [reply thoth/common/openshift.py]: Sorry, I didn't get that indeed :) Let's discuss it on Monday :)

Comment (pacospace) [reply thoth/common/openshift.py]: Updated, as discussed. Thanks! :beers:

**Comment by pacospace** [comment]
recheck

**Pull request merged**